routes/movies.routes.js

```diff
@@ -70,12 +70,40 @@ router.post("/movies/:moviesId/delete", (req, res, next) => {
 })
 
 
+
 router.get("/movies/:moviesId/edit", (req, res, next) => {
-    Movie.findById(req.params.moviesId)
-    Celebrity.find().populate("cast")
+    const movieInfo = Movie.findById(req.params.moviesId).populate("cast")
+    const celebrityInfo = Celebrity.find()
+
     .then(() => {
-        res.render("movies/edit-movie", )
+        
+
+        res.render("movies/edit-movie", {movieInfo, celebrityInfo})
     })
```

Comment:
Another clue is here. These are asynchronous functions, so you cannot know the order in which  they'll be executed. So think about using promises, how to make sure it is executed in the order that you want it to be.